contributors/design-proposals/componentconfig-dump.md

```diff
@@ -0,0 +1,164 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
```

Comment:
please remove this UNVERSIONED_WARNING header - it is obsolete.  Where did you copy this from?